File: QA/viasoft.qualidade.rnc-frontend/apps/rnc/src/api-clients/Estoque-Pedido-Venda-Estoque-Locais/model/estoque-pedido-venda-estoque-local-view-output.ts
export class EstoquePedidoVendaEstoqueLocalViewOutput {
  public id: string;
  public legacyId: number;
  public idEstoquePedidoVenda: string;
  public idEstoqueLocal: string;
  public isLocalBloquearMovimentacao: boolean;
  public numeroLote: string;
  public codigoLocal: number;
  public dataFabricacao: Date | null;
  public dataValidade: Date | null;
  public idProduto: string;
  public idPedido: string;
  public quantidade: number;
}

File: viasoft.qualidade.rnc.gateway/Viasoft.Qualidade.RNC.Gateway.Host/Proxies/LegacyLogistica/EstoquePedidoVendaEstoqueLocalViews/Dtos/EstoquePedidoVendaEstoqueLocalViewOutput.cs
using System;

namespace Viasoft.Qualidade.RNC.Gateway.Host.Proxies.LegacyLogistica.EstoquePedidoVendaEstoqueLocalViews.Dtos;

public class EstoquePedidoVendaEstoqueLocalViewOutput
{   
    public Guid Id { get; set; }
    public int LegacyId { get; set; }
    public Guid IdEstoquePedidoVenda { get; set; }
    public Guid IdEstoqueLocal { get; set; }
    public bool IsLocalBloquearMovimentacao { get; set; }
    public string NumeroLote { get; set; }
    public int CodigoLocal { get; set; }
    public DateTime? DataFabricacao { get; set; }
    public DateTime? DataValidade { get; set; }
    public Guid IdProduto { get; set; }
    public Guid IdPedido { get; set; }
    public decimal Quantidade { get; set; }
    public int NumeroOdf { get; set; }
}